build.gradle

```diff
@@ -104,7 +104,7 @@ dependencies {
     testCompile "org.spockframework:spock-spring"
     testCompile "org.seleniumhq.selenium:selenium-firefox-driver:$seleniumVersion"
     testCompile "org.seleniumhq.selenium:selenium-support:$seleniumVersion"
-    testCompile "org.seleniumhq.selenium:selenium-htmlunit-driver:$seleniumVersion"
+    testCompile "org.seleniumhq.selenium:selenium-htmlunit-driver:2.52.0"

```

Comment:
Hmm, I didn't know about a separation. In that case maybe this version (released in parallel to Selenium 2.53.0) could be used? https://github.com/SeleniumHQ/htmlunit-driver/releases/tag/2.20